src/applications/PerfusionAlignment.h

```diff
@@ -160,6 +161,13 @@ class PerfusionAlignment
     return std::make_pair(stdDevImage, stdDevVector);
   }
 
+  /**
+  \brief fill the chart with supplied perfusion alignment data
+  \param chart pointer to vtkChartXY that will be filled
+  \param map pointer to map of data name to pointer of vector of data that needs to be plotted
+  */
+  void Createchart(vtkChartXY* chart, std::map<std::string, std::vector<float>*>* map);
```

Comment:
```suggestion
  void Createchart(vtkChartXY* chart, std::map<std::string, std::vector<float>> map);
```

Reply:
No change needed. What we have is correct and avoids duplication of memory.